pkg/kinflate/resource/util.go

```diff
@@ -107,11 +109,14 @@ func MergeWithOverride(rcs ...ResourceCollection) (ResourceCollection, error) {
 				case "", constants.CreateBehavior:
 					return nil, fmt.Errorf("Create an existing gvkn %#v is not allowed", gvkn)
 				case constants.ReplaceBehavior:
+					glog.V(4).Infof("Replace object %v by %v", all[gvkn].Data.Object, obj.Data.Object)
 					obj.replace(all[gvkn])
 					all[gvkn] = obj
 				case constants.MergeBehavior:
+					glog.V(4).Infof("Merge object %v with %v", all[gvkn].Data.Object, obj.Data.Object)
 					obj.merge(all[gvkn])
 					all[gvkn] = obj
+					glog.V(4).Infof("The merged object is %v", obj.Data.Object)
```

Comment:
It should be `all[gvkn].Data.Object` here.

Reply:
Updated, but they are actually the same by this step.